levelObjects/Actions.md

```diff
@@ -20,10 +20,12 @@ If the value is decremented to zero after the action was executed, the object wh
 
 **Transport Player Action Details** (16 byte)
 
-    0000  int32      Target X
-    0004  int32      Target Y
-    0008  int32      Unknown
-    000C  int32      Unknown
+	0000  int32      Target X
+	0004  int32      Target Y
+	0008  int16      Height
+	000A  int16      Pitch
+	000C  int16      Yaw
+	000E  int16      Roll
```

Comment:
It's not Pitch/Yaw/Roll - orientation is always maintained by a transport.
Instead, the last int32 is capable of specifying target level! I haven't followed the even chain, though I believe this is how death is handled on two of the groves.